src/Renderer/ThreeExtended/OBB.js

```diff
@@ -98,6 +98,24 @@ OBB.prototype._cPointsWorld = function _cPointsWorld(points) {
     return points;
 };
 
+/**
+ * Determines if the sphere is above the XY space of the box
+ *
+ * @param      {Sphere}   sphere  The sphere
+ * @return     {boolean}  True if sphere is above the XY space of the box, False otherwise.
+ */
+OBB.prototype.isSphereAboveXYBox = function isSphereAboveXYBox(sphere) {
+    // get obb closest point to sphere center by clamping
+    var x = Math.max(this.box3D.min.x, Math.min(sphere.position.x, this.box3D.max.x));
```

Comment:
You don't need to use `this.matrixWorld` here?

Reply:
(and please use const where possible)

Reply:
And please could you add a unit test to cover this feature?

Reply:
yes, it's better, I move transform local position here

> And please could you add a unit test to cover this feature?

I don't have time for the moment

Reply:
I add the unit test